decidim/decidim
# Fix Creating a Conference speaker with an attachment will throw Module::DelegationError

#### :tophat: What? Why?
Creating a Conference speaker with an attachment will throw Module::DelegationError

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #7190

#### Testing

1. Go to Conference speakers area 
2. Click on Add new speaker
3. Fill in all the details 
4. Attach  an image 
5. See the error 
6. Apply the patch
7. See the error is on the form

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!
